src/workloads/execution/MultiDeletesDocByDoc.yml

```diff
@@ -0,0 +1,27 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+  Runs a workload that inserts and then deletes a large range of documents on one collection while
+  performing writes on another collection in the same database. Serves as a base line for comparing
+  legacy document by document deletes versus efficiently batched multi deletes.
+
+# Increase the maxPoolSize to enable faster initial data population
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 300
```

Comment:
This was done to speed up the inserts significantly. When I added 2 separate MonotonicSingleLoader phases, I had to raise it from 200 - 300